das_client/app/lib/pages/journey/break_load_slip/widgets/break_load_slip_special_restrictions.dart

```diff
@@ -0,0 +1,97 @@
+import 'package:app/i18n/i18n.dart';
+import 'package:app/pages/journey/break_load_slip/widgets/break_load_slip_data_row.dart';
+import 'package:app/widgets/assets.dart';
+import 'package:app/widgets/das_colors.dart';
+import 'package:app/widgets/das_text_styles.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:formation/component.dart';
+import 'package:sbb_design_system_mobile/sbb_design_system_mobile.dart';
+
+class BreakLoadSlipSpecialRestrictions extends StatelessWidget {
+  const BreakLoadSlipSpecialRestrictions({required this.formationRun, super.key});
+
+  final FormationRun formationRun;
+
+  @override
+  Widget build(BuildContext context) {
+    return SBBGroup(
+      child: Stack(
+        children: [
+          _specialIndicators(),
+          _content(context),
+        ],
+      ),
+    );
+  }
+
+  Row _specialIndicators() {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.end,
+      children: [
+        if (formationRun.simTrain) _indicator(AppAssets.iconSimZug, DASColors.simTrain),
+        if (formationRun.dangerousGoods) _indicator(AppAssets.iconSignExclamationPoint, SBBColors.peach),
+      ],
+    );
+  }
+
+  Padding _content(BuildContext context) {
+    return Padding(
```

Comment:
Der Grundaufbau mit der Tabelle wird mehrfach verwendet und sieht immer gleich aus.

```dart
Padding(
      padding: const EdgeInsets.all(sbbDefaultSpacing * 0.5),
      child: Column(
        mainAxisSize: MainAxisSize.min,
        children: [
          BreakLoadSlipDataRow(),
          BreakLoadSlipDataRow(),
          ....
         ]),
)
```

Evtl. kann man hier wie bei der DASTable ein generelles Widget machen, was später auch wiederverwendet werden kann. Name für Widget/Row kommen mir noch keine gute in den Sinn. `KeyValueTable`, `SingleValueTable`, `TextDataTable`

```dart
KeyValueTable(
        rows: [
          DataRow(),
          DataRow(),
          ....
         ]),
)
```